pepper-apis/studybuilder-cli/studies/lms/i18n/es.json

```diff
@@ -41,6 +41,34 @@
       "required_hint": "Please choose a province"
     }
   },
+
+  "somatic_results": {
```

Comment:
Supposed to be English in the Spanish translation file?

Reply:
Its not used currently but when we get there we send this file to translator and will get spanish translations for this english text.